app/javascript/react/components/DonutReviewPhotoFieldTile.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const DonutReviewPhotoField = (props) => {
+  return(
+    <label  className="form-label">{props.label}
+      <input
+        name={props.name}
+        type='text'
+        value={props.content}
+        onChange={props.handleNewDonutReviewPhotoField}
+      />
+    </label>
```

Comment:
For photos in React, you will have to use React Dropzone. This field probably does not work at the moment :/ You can find an article on dropzone in the readings for this week. 
Otherwise, I dont know why you would need a photo for a review, so I would just remove this component